static/styles/compose.scss

```diff
@@ -468,14 +468,14 @@ a.message-control-button {
 
 a#markdown_preview {
     margin-left: 2px;
-    color: hsl(0, 0%, 47%);
+    color: inherit;
 }
 
 a#undo_markdown_preview {
     text-decoration: none;
     position: relative;
     font-size: 15px;
-    color: hsl(0, 0%, 47%);
+    color: inherit;
```

Comment:
Do we need `color: inherit` here, or could we just delete the `color` CSS altogether and just use the general `body.night_mode a:hover` rule?